services/tables/src/main/java/com/linkedin/openhouse/tables/config/TablesMvcConfigurer.java

```diff
@@ -19,29 +23,45 @@
  */
 @Configuration
 public class TablesMvcConfigurer implements WebMvcConfigurer {
-
   @Autowired private ClusterProperties clusterProperties;
+  OpenAPI openAPI =
```

Comment:
consolidating SwaggerConfig into this file, for benefit of having more centralized config. config across multiple files can lead to hard to trace keys/values when debugging. 

using this as inspiration: https://github.com/davidivkovic/uber-rides/blob/772317ce1c0baea8fd54f6e2b49c1a52e58a3853/src/services/rides/src/main/java/com/uber/rides/security/Config.java#L51

Reply:
is there other services' configurer using this `SwaggerConfig` ?

Reply:
ok I see this being annotated as a bean in the following. 
I believe the original intention to have SwaggerConfig separated is for sharing among different services

Reply:
there is no other service using this swagger config -- it is suited only for tables API / controller.

ideally we instantiate SwaggerConfig in global config. then Autowired it to each controller for extensions to e.g. the API info title ".title("OpenHouse Tables APIs") or .title("OpenHouse Jobs APIs")

but right now the APIs don't share anything in common so there is no benefit to converging them